pkg/configurator/client.go

```diff
@@ -254,7 +254,7 @@ func (c *Client) getConfigMap() *osmConfig {
 	}
 
 	if !exists {
-		log.Error().Msgf("ConfigMap %s does not exist in cache", configMapCacheKey)
+		log.Warn().Msgf("ConfigMap %s does not exist. Default config values will be used.", configMapCacheKey)
```

Comment:
Is this error seen only when the cache is not yet initialized? Because the ConfigMap should always exist after install?

Reply:
It should, but it could be deleted or if you are running chunks of this code base you may run into this.
I saw this message while running tests (not super relevant to this area of the code base).
Warning is a good thing to know - ConfigMap should generally exist. Definitely not an Error.  Seeing an error sends me on a wild goose chase to fix something that does not need fixing ;)